scripts/gen_manifests.sh

```diff
@@ -21,7 +21,7 @@ echo -e "\nquarkus.flyway.migrate-at-start=true" >> application.properties
 
 # TODO Update to use --skip-namespace when the following is released
 # https://github.com/apache/incubator-kie-tools/pull/2136
-kn workflow gen-manifest --namespace ""
+kn-workflow gen-manifest --namespace ""
```

Comment:
line 15
`command -v kn` should be updated or removed

Reply:
actually, it doesn't since we include both `kn` and `kn-workflow` in the same image.
so perhaps add a check that `kn-workflow` is available?

Reply:
at the moment I left kn in the image though